examples/tools/use-cases/custom_llm_tool_showcase/README.md

```diff
@@ -0,0 +1,36 @@
+# Flow with custom_llm tool
+This is a flow demonstrating how to use a `custom_llm` tool, `custom_llm` tool allows users to connect to a customized large language model which is fine-tuned by tool developer. 
+
+Tools used in this flow:
+- `custom_llm` Tool
+
+Connections used in this flow:
+- None
+
+## Prerequisites
+
+Install promptflow sdk and other dependencies:
+```bash
+pip install -r requirements.txt
+```
+
+## Setup connection
+Prepare your Azure Open AI resource follow this [instruction](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) and get your `api_key` if you don't have one.
```

Comment:
why do we need to set up connection? Since line 7 says no

Reply:
This is a fake connection, a similar flow is as following, thanks:
https://github.com/microsoft/promptflow/blob/404bce1467bc47f7e2732f9a020c43b3d48cd49c/examples/flows/standard/basic-with-connection/README.md?plain=1#L29
![image](https://github.com/microsoft/promptflow/assets/95729303/d79fec9d-991c-4e0f-943c-6e3aa143b201)

Reply:
Please refer the readme in this PR to refine the customconnection part https://github.com/microsoft/promptflow/pull/921

Reply:
Changed, thanks.